assets/prototype/domain/shared/dateRangeInput/DateRangePicker.js

```diff
@@ -27,7 +27,7 @@ const DateRangePicker = ({ onFieldUpdate, range, setRange }) => {
 	const appliedProps = {
 		allowSingleDayRange: true,
 		contiguousCalendarMonths: false,
-		dateRange: range,
+		value: range,
```

Comment:
This is a 1-line fix for #1928, that's why I didn't created a separate PR just for that.
Here is the fix:

![Peek 2019-12-06 14-19](https://user-images.githubusercontent.com/1477453/70322683-6b405b80-1833-11ea-93c9-299e05e3d48a.gif)

Reply:
HA!  so much for my 8 hour estimate :\

Reply:
I got misled before by the example set by blueprint: https://github.com/palantir/blueprint/blob/develop/packages/docs-app/src/examples/datetime-examples/dateRangePickerExample.tsx#L36